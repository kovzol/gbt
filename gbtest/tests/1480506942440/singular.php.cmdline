#!/bin/sh
cd tests/1480506942440
cat singular.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/Singular > singular.php.out 2>singular.php.err