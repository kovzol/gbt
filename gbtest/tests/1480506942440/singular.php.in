ring r=0,(x,y),lp;ideal I=x^2+x*y-1,2*x*y+3*x,x^3; option(redSB); groebner(I);