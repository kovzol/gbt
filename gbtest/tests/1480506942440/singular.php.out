                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /   version 4.0.3
                                                           0<
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \   Jan 2016
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \  Debian 4.0.3+ds-1
_[1]=1
> Auf Wiedersehen.
